README

```diff
@@ -16,5 +16,5 @@ William Hart -- 16 Jan 2011.
 
 License
 -------
-This version is distributed under LGPL3 (GNU Library Public License,
+This version is distributed under LGPL 2.1 (GNU Library Public License,
 also named GNU Lesser Public License). See the LICENSE file.
```

Comment:
Apologies. Anything else?

On Thu, Apr 28, 2016 at 3:06 PM Fredrik Johansson <notifications@github.com>
wrote:

> In README <https://github.com/wbhart/flint2/pull/260#discussion_r61423115>
> :
>
> > @@ -16,5 +16,5 @@ William Hart -- 16 Jan 2011.
> >
> >  License
> >  -------
> > -This version is distributed under LGPL3 (GNU Library Public License,
> > +This version is distributed under LGPL 2.1 (GNU Library Public License,
> >  also named GNU Lesser Public License). See the LICENSE file.
>
> It is named the "GNU Lesser General Public License".
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/wbhart/flint2/pull/260/files/d5560c675e0d469059e742a6ce6692c4cc8c69dc..9a9fcdc111f962928fb5cca6d05bdae70f009398#r61423115>
>